src/CalculatorUITestFramework/WinAppDriver.cs

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium.Appium;
+using OpenQA.Selenium.Appium.Windows;
+using System;
+using System.Diagnostics;
+
+namespace CalculatorUITestFramework
+{
+    public sealed class WinAppDriver
+    {
+        private WindowsDriverLocalService windowsDriverService = null;
+        private const string calculatorAppId = "Microsoft.WindowsCalculator.Dev_8wekyb3d8bbwe!App";
+        private static WinAppDriver instance = null;
```

Comment:
In this case, it's appropriate to collapse the singleton boilerplate into a one-liner:

public static WinAppDriver Instance { get; } => new WinAppDriver();

Reply:
I think this is creating a new instance of WinAppDriver each time Instance is called. I tried this and it caused a bunch of null reference exceptions. So I decided to stay with the original Singleton implementation.

Reply:
I mistyped the comment above. I should have said:

public static WinAppDriver Instance { get; } = new WinAppDriver();

...which will only call the constructor once.